src/Views/Admin/Pages/Setup.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * Setup Page class
+ *
+ * @package Give
+ */
+
+namespace Give\Views\Admin\Pages;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Organizes WordPress actions for the Setup Page submenu.
+ *
+ * @since 2.8.0
+ */
+class Setup {
+
+	/**
+	 * Initialize Reports Admin page
+	 *
+	 * @since 2.8.0
+	 */
+	public function init() {
+		add_action( 'admin_init', [ $this, 'redirectDonationsToSetup' ] );
+		add_action( 'admin_menu', [ $this, 'add_page' ] );
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_scripts' ] );
+	}
+
+	public function __construct() {
```

Comment:
@kakshak Remove this because it does nothing

Reply:
@ravinderk sure. This was copy/pasted from an existing example in the codebase, but I can make that change.